.test_director

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
 set -xeuo pipefail
 
+VIRTUAL_ENV_DISABLE_PROMPT=1
+
+if [ ! -d venv ]; then
+    virtualenv venv
```

Comment:
With `libselinux-python` installed, we should use `virtualenv --system-site-packages venv` to ensure we get the SELinux bindings available.